libs/appproxy/appproxy_test.go

```diff
@@ -134,9 +134,18 @@ func TestProxyHandleWebSocket(t *testing.T) {
 
 	_, _, err = conn.ReadMessage()
 	require.Error(t, err)
-	if !strings.Contains(err.Error(), "websocket: close 1006 (abnormal closure)") &&
-		!strings.Contains(err.Error(), "An established connection was aborted by the software in your host machine") &&
-		!strings.Contains(err.Error(), "connection reset by peer") {
-		t.Errorf("Expected abnormal closure, An established connection was aborted, or connection reset by peer, got %s", err)
+	potentialErrMessages := []string{
+		"websocket: close 1006 (abnormal closure)",
+		"An established connection was aborted by the software in your host machine",
+		"connection reset by peer",
+		"An existing connection was forcibly closed by the remote host",
 	}
+	for _, msg := range potentialErrMessages {
+		if strings.Contains(err.Error(), msg) {
+			return
```

Comment:
nit: return works because it's the last check but prevents adding more code at the bottom. I'd extract error message checking into separate function